package.json

```diff
@@ -30,6 +30,9 @@
     "scripts_comments": {
         "build:jitsi": "tchap don't need that, deactivate?"
     },
+    "matrix_i18n_extra_translation_funcs": [
```

Comment:
what is it? it seems interesting

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/7377

Reply:
I haven't found other occurrence of `matrix_i18n_extra_translation_funcs`  in the matrix code, but it looks like it allows us to use the new error generating method `UserFriendlyError` which we discussed in a previous comment. And given the array in the package.json, we should now be able to use more than one error generating method :)